gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -539,8 +539,10 @@ export function GeneralSettings() {
                   variant="toggle"
                   outlined
                   control={control}
-                  name="toggles.viveEmulation"
-                  label={t('settings-general-fk_settings-vive_emulation-label')}
+                  name="toggles,viveEmulation"
```

Comment:
I am very sure that is a wrong one, lol

Reply:
did that recently get changed to a comma? I'll try tho